### PR TITLE
rust: add rustdoc descriptions to the public API and tiny improvements for crates.io

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: bindings/rust/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Fizzy
```
Comment (axic) [inline bindings/rust/README.md:1]: This is what will be visible on crates.io

**Review by chfast — CHANGES_REQUESTED** [review]
Overlapped with #587

**Comment by axic** [comment]
I know, I would merge this first.

**Comment by chfast** [comment]
Please combine these 2 PRs to avoid the noise.

**Comment by axic** [comment]
What noise?

File: bindings/rust/src/lib.rs [hunk 2]
```diff
@@ -2,10 +2,15 @@
 // Copyright 2019-2020 The Fizzy Authors.
 // SPDX-License-Identifier: Apache-2.0
 
+//! This is a Rust interface to [Fizzy](https://github.com/wasmx/fizzy), a WebAssembly virtual machine.
```
Comment (chfast) [inline bindings/rust/src/lib.rs:5]: What's the difference between `//!` and `///`?

**Review by chfast — APPROVED** [review]

File: bindings/rust/src/lib.rs (hunk 2 above)
Comment (axic) [reply bindings/rust/src/lib.rs:5]: `//!` is the description shown on the top of docs.rust.rs, while `///` is the doxygen compatible description for syntactical elements.

**Pull request merged**